Makefile

```diff
@@ -219,7 +219,9 @@ test-integration-no-cluster:
 # For example: "PODMAN_EXEC_NODES=5".
 .PHONY: test-integration-podman
 test-integration-podman:
-	$(RUN_GINKGO) $(GINKGO_FLAGS_PODMAN) --junit-report="test-integration-podman.xml" --label-filter=podman tests/integration
+	# TODO: to remove
+	$(RUN_GINKGO) $(GINKGO_FLAGS_PODMAN) --junit-report="test-integration-podman.xml" --label-filter="podman && odolist && describecomponent" tests/integration
```

Comment:
Condition should be `podman && (odolist || describecomponent)`. But actually, I would suggest we run each of them individually, to make sure we don't have unexpected issues (especially with `odo list` tests).

```sh
$(RUN_GINKGO) $(GINKGO_FLAGS_PODMAN) --junit-report="test-integration-podman.xml" --label-filter="podman && odolist" tests/integration
$(RUN_GINKGO) $(GINKGO_FLAGS_PODMAN) --junit-report="test-integration-podman.xml" --label-filter="podman && describecomponent" tests/integration
$(RUN_GINKGO) $(GINKGO_FLAGS_PODMAN) --junit-report="test-integration-podman.xml" --label-filter="podman && !(odolist || describecomponent)" tests/integration
```

Also, the job took 46m with 5 parallel nodes, which is way too long for me.
Can you also revert back to 10 parallel nodes, so that we can see what happens?

Reply:
Yeah, Will do